lib/src/alert_dialog/show_alert_dialog.dart

```diff
@@ -30,11 +30,13 @@ Future<T?> showAlertDialog<T>({
   Widget? macOSApplicationIcon,
   RouteSettings? routeSettings,
 }) {
-  final navigator = Navigator.of(
-    context,
-    rootNavigator: useRootNavigator,
-  );
-  void pop(T? key) => navigator.pop(key);
+  NavigatorState navigator(BuildContext context) {
+    return Navigator.of(
+      context,
+      rootNavigator: useRootNavigator,
+    );
+  }
```

Comment:
I think this is better.

```dart
  void pop({required BuildContext context, required T? key}) => Navigator.of(
        context,
        rootNavigator: useRootNavigator,
      ).pop(key);
```

```dart
onPressed: (key) => pop(context: context, key: key),
```

(I just realized that there was no point in separating the navigator and pop methods originally.)